docs/source/user-guide/tasks/manage-channels.rst

```diff
@@ -99,7 +63,30 @@ Conda now has an equivalent command::
 
   conda config --prepend channels new_channel
 
-Conda also now has a command that adds the new channel to the 
+Conda also now has a command that adds the new channel to the
 bottom of the channel list, making it the lowest priority::
 
   conda config --append channels new_channel
+
+Strict channel priority
+=======================
+
+Conda has a strict channel priority feature. Details about it
+can be seen by typing ``conda config --describe channel_priority``.
+
+.. code-block:: none
+
+    channel_priority (ChannelPriority)
+    Accepts values of 'strict', 'flexible', and 'disabled'. The default
+    value is 'flexible'. With strict channel priority, packages in lower
+    priority channels are not considered if a package with the same name
+    appears in a higher priority channel. With flexible channel priority,
+    the solver may reach into lower priority channels to fulfill
+    dependencies, rather than raising an unsatisfiable error. With channel
+    priority disabled, package version takes precedence, and the
+    configured priority of channels is used only to break ties. In
+    previous versions of conda, this parameter was configured as either
+    True or False. True is now an alias to 'flexible'.
```

Comment:
Strict channel priority can dramatically speed up conda operations and also reduce package incompatibility problems.  We recommend it as a default. However, it may break old environment files, so we plan to delay making it conda's out-of-the-box default until the next major version bump, conda 5.0.